cmd/flags_node.go

```diff
@@ -9,34 +9,56 @@ import (
 	flag "github.com/spf13/pflag"
 
 	"github.com/celestiaorg/celestia-node/node"
+	"github.com/celestiaorg/celestia-node/params"
 )
 
 var (
-	nodeStoreFlag  = "node.store"
-	nodeConfigFlag = "node.config"
+	nodeStoreFlag   = "node.store"
+	nodeConfigFlag  = "node.config"
+	nodeNetworkFlag = "node.network"
 )
 
 // NodeFlags gives a set of hardcoded Node package flags.
-func NodeFlags(tp node.Type) *flag.FlagSet {
+func NodeFlags() *flag.FlagSet {
 	flags := &flag.FlagSet{}
 
 	flags.String(
 		nodeStoreFlag,
-		fmt.Sprintf("~/.celestia-%s", strings.ToLower(tp.String())),
+		"",
```

Comment:
Why remove the default? It is convenient, especially for users printing out help for the CLI

Reply:
We do this for a similar reason as above: When a user overrides the default network, it needs to be placed in that string. If we create a default, we have to override it anyways when parsing.

Reply:
It's not an issue when you don't override params pkg

Reply:
Nothing is an issue when you don't do the thing that causes the issue

Reply:
What do you mean?